zc_common/remote_resource/relations.py

```diff
@@ -9,14 +9,42 @@
 
 
 class RemoteResourceField(ResourceRelatedField):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, related_resource_path=None, **kwargs):
         if 'model' not in kwargs:
             kwargs['model'] = RemoteResource
         if not kwargs.get('read_only', None):
             # The queryset is required to be not None, but not used
             #   due to the overriding of the methods below.
             kwargs['queryset'] = {}
-        super(RemoteResourceField, self).__init__(*args, **kwargs)
+
+        if related_resource_path is None:
+            raise NameError('related_resource_path parameter must be provided')
+
+        self.related_resource_path = related_resource_path
+
+        super(RemoteResourceField, self).__init__(**kwargs)
+
+    def get_links(self, obj=None, lookup_field='pk'):
```

Comment:
This looks pretty good. I have no opinions on how the links get generated -- we could even make it so every field has to implement it or have it be like default fields in django models (where they take a static value or callable function)